configure.ac

```diff
@@ -5,13 +5,13 @@ dnl Licensed under LGPL. See file COPYING.
 dnl
 dnl ref: http://www.gnu.org/software/autoconf/manual/autoconf.info.gz
 dnl
-AC_PREREQ(2.57)
+AC_PREREQ([2.71])
```

Comment:
I dont want this requirement.  I thought you were going to revert these changes to configure.ac?